charts/mysql-alerts/remedy.md

```diff
@@ -37,9 +37,29 @@
 - #### AppPhaseNotReady
   - Contact AppsCode team
 - #### AppPhaseCritical
-  - If any `MySQLOpsRequest` is ongoing on same database, Wait until it completes.
-  - If some nodes of the MySQL group are not `Up`, Try restarting those nodes one at a time.
-  - Contact AppsCode team if this persists for more than 30 minutes.
+    - If the Database enters a `Critical` phase and the pod logs show the error `Error_code: MY-013121`, it's likely that the relay log and binlog are corrupted.
```

Comment:
`the source's binary log is corrupted` Include this log too.  The error code does not get the proper attention (from user perspective)